provider/aws/config.go

```diff
@@ -84,8 +83,10 @@ func newV2Config(awsConfig AWSSessionConfig) (awsv2.Config, error) {
 		config.WithRetryer(func() awsv2.Retryer {
 			return retry.AddWithMaxAttempts(retry.NewStandard(), awsConfig.APIRetries)
 		}),
-		config.WithHTTPClient(extdnshttp.NewInstrumentedClient(&http.Client{})),
+		config.WithHTTPClient(awshttp.NewBuildableClient()),
```

Comment:
I did some research, this is a default client. We could keep it, to make it explicit, more a question where or not it will work correctly without it